app/boards/arm/sparkfun_micromod_nrf52840/sparkfun_micromod_nrf52840.dts

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) 2021 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+#include "sparkfun_micromod.dtsi"
+
+
+/ {
+	model = "Sparkfun MicroMod nrf52840";
+	compatible = "sparkfun,micromod-gpio";
+
+	chosen {
+		zephyr,code-partition = &code_partition;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		blue_led: led_0 {
+			gpios = <&gpio0 13 GPIO_ACTIVE_HIGH>;
+			label = "Blue LED";
+		};
+	};
+};
+
+&adc {
+	status = "okay";
+};
+
+&gpiote {
+	status = "okay";
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&usbd {
+	status = "okay";
+};
+
+
+
+&spi1 {
+	compatible = "nordic,nrf-spim";
+	/* Cannot be used together with i2c0. */
+	status = "okay";
```

Comment:
i wouldn't enable this by default.

Reply:
@petejohanson why should this not be enabled by default? It is the dedicated SPI on the general pinout. I suppose for keyboards you need all the GPIO you can get though, is that why?

Reply:
In general, yeah, alternate functions for pins are usually only enabled for UART at most in Zephyr, by convention.

Now, the micro mod certainly has more gpio than other format... But I think this still holds.

Same reason the nano SPI and I2C aren't enabled by default.